src/app/components/give-clue/give-clue.component.ts

```diff
@@ -39,20 +40,24 @@ export class GiveClueComponent implements OnDestroy {
   ) {}
 
   ngOnInit() {
-    this.proposedClue$ = this.clueService.getProposedClue(this.game.id)
-    this.proposedClue$.subscribe(proposedClue => {
-      this.proposedClue = proposedClue;
-      if (this.proposedClue && !this.isMyTurn) {
-        this.alertTimerId = setInterval(() => {
-          if (this.proposedClue && !this.isMyTurn){
-            this.soundService.askFirstAlert();
-          } else {
-            clearInterval(this.alertTimerId);
-          }
-        }, 3000);
-        this.soundService.askFirstAlert();
-      }
-    });
+    this.proposedClue$ =
+        this.clueService.getProposedClue(this.game.id)
+            .pipe(tap(proposedClue => {
```

Comment:
We should use tap instead of subscribe where possible to prevent memory leaks. The `async` pipe (used in the template) automatically handles subscribing/unsubscribing so we don't have to do it manually.